utils/measure/measure.py

```diff
@@ -276,7 +347,32 @@ def start(self):
             self.gzip_csv(csv_file_path)
 
     def should_resume(self, csv_file_path: str) -> bool:
-        """Check whether we are able to resume a previous measurement session"""
+        """This method checks if a CSV file already exists for the current color mode.
+        
+        If so, it asks the user if he wants to resume measurements or start over.
```

Comment:
This method don't asks the user yet, only checks if it can resume. So it is incorrect in that sense. But I don't mind leaving it, what do you think?

Reply:
I mean, it calls inquirer.confirm(), so it kinda asks the user, doesn't it?

Reply:
Nevermind, you are completely right.